test/unit/models/users.test.js

```diff
@@ -115,6 +115,16 @@ describe("users", function () {
       const { user } = await users.fetchUser({ githubUsername });
       expect(user).to.haveOwnProperty("github_created_at");
     });
+
+    it("it should filter out the id field while updating profileDiff", async function () {
+      const userData = userDataArray[0];
+      const { userId } = await users.addOrUpdate(userData);
+      const profileDiff = { id: "random-id", diff: "random-diff" };
```

Comment:
we should test with partial data where only a subset of fields in the user profile is updated.

Reply:
Doing this in Line 119 - Line 127 where only 2 fields are updated using profileDiff